core/services/vrf/vrf_solidity_crosscheck_test.go

```diff
@@ -359,8 +359,8 @@ func TestVRF_MarshalProof(t *testing.T) {
 		sk := randomScalar(t, r)
 		skNum := secp256k1.ToInt(sk)
 		nonce := randomScalar(t, r)
-		seed := randomUint256(t, r)
-		proof, err := vrf.GenerateProofWithNonce(skNum, seed, secp256k1.ToInt(nonce))
+		sd := randomUint256(t, r)
```

Comment:
why rename this var?

Reply:
Go shadow complained about it.

I have to assume something got merged such that it is also broken in develop.